tests/tck/features/MatchAcceptance2.feature

```diff
@@ -1387,6 +1387,7 @@ Feature: MatchAcceptance2
       | (:Artist:B) | (:Artist:C) |
     And no side effects
 
+@skip
```

Comment:
So now with this PR fix, we fail this tck test?

Reply:
Yes, see description at https://github.com/RedisGraph/RedisGraph/pull/913#discussion_r379663012